jobs/env/ci-kubernetes-e2e-gci-gce-netd.env

```diff
@@ -0,0 +1,123 @@
+### job-env
+KUBE_CUSTOM_NETD_YAML="""
```

Comment:
@BenTheElder @krzyzacy  the scenarios/kubernetes_e2e.py change is for this. the Current code cannot handle var values that contains more than one line

Reply:
as I mentioned, this yaml blob should not be a env value, it should be a path to your yaml manifest and you can read it from the path